.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: codeql
+
+on:
+  push:
+    branches:
+    - master
+    - v1.9
```

Comment:
late to the party, but maybe `v*.*` here? seems pretty easy to forget to update this file after each release.